src/routes/landing.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+class Landing extends React.Component {
+  render() {
+    return (
+      <View
+        style={styles.landing}>
+        <Text>Landing</Text>
+        <Button
+          title='Go To Search'
+          onPress={() => this.props.navigation.navigate('Search')}
```

Comment:
This causes the "up" button to appear on the top corner. This may not be ideal if you have a nav bar at the bottom.

Reply:
Yes, I haven't changed any of the layout compared to the examples I followed, so we can tweak that based on the wire-frames we get. I just wanted to get something up here for us to start with.